src/topp/IDFileConverter.cpp

```diff
@@ -205,10 +206,10 @@ class TOPPIDFileConverter :
     registerInputFile_("in", "<path/file>", "",
                        "Input file or directory containing the data to convert. This may be:\n"
                        "- a single file in a multi-purpose XML format (pepXML, protXML, idXML, mzid),\n"
-                       "- a single file in a search engine-specific format (Mascot: mascotXML, OMSSA: omssaXML, X! Tandem: xml, Percolator: psms),\n"
+                       "- a single file in a search engine-specific format (Mascot: mascotXML, OMSSA: omssaXML, X! Tandem: xml, Percolator: psms, xQuest: xquest.xml),\n"
```

Comment:
just for clarity, should we prepend file extensions with a dot, i.e. write `.mascotXML, .... .xquest.xml` ?

Reply:
Yes, I agree. Will do that.